### ISSUE TITLE
Facebook Login / API Change 

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-mobile#67]
Per the e-mail I forwarded, the Facebook change requires our action by August 6th. We have time, but we may want to address this as part of the fix for #66

**Comment by petervojtek** [comment]
blocked by #66

**Comment by petervojtek** [comment]
We call the facebook graph API from two places in code: [1](https://github.com/PowerlineApp/powerline-mobile/blob/develop/www/js/services/session.js#L206) and [2](https://github.com/PowerlineApp/powerline-mobile/blob/develop/www/js/services/session.js#L309)

The solution seems to be easy: https://github.com/Wizcorp/phonegap-facebook-plugin/issues/1284#issuecomment-227833640

**Comment by jterps08** [comment]
Wow - looks like Wizcorp needs to get on their game. 

Yes,
(1) Registration information is filled in with assistance from Facebook
(2) My Influences suggests friends from Facebook

In the future, we will want to allow a user to share specific content to Facebook (allow the app to post to Facebook), but not now.

**Comment by petervojtek** [comment]
it is possible the commit I made (upgrade facebook plugin version) is all we need to do, however I will be able to confirm that only after #66 is resolved. Also check [this pull request](https://github.com/PowerlineApp/powerline-mobile/pull/51/commits/247ab29cdef946e3e6ec940fd176230681185566)

**Comment by jterps08** [comment]
Just want to confirm that we are on track to change this. The FB API v2.0 will be deprecated this week. v2.7 needs to be called instead.

**Comment by petervojtek** [comment]
ok, it seems all works well, because it is already past the deadline and facebook login works.

> Powerline has been making recent API calls to Graph API v2.0, which will reach the end of the 2-year deprecation window on Monday, August 8, 2016.

However, I am not sure which API version we are using right now. the best way to determine it is to take a look in Graph API update tool, however I do not have right to access the powerline facebook app:
https://developers.facebook.com/tools/api_versioning/244141112405524

**Comment by jterps08** [comment]
Everyone is automatically migrated to v2.1... 

That link you provided is good, unfortunately we do not make enough API calls for it to tell us anything. 

According to @igaponov, the backend has been updated to access the latest version of graph api, but the frontend has its own separate way of accessing the API too.. Either way this item seems closed now.
